lib/model/query/audits.js

```diff
@@ -150,11 +150,19 @@ const getBySubmissionId = (submissionId, options) => ({ all }) => _getBySubmissi
 // There is a separate query below to assemble full submission details for non-deleted
 // submissions, but it was far too slow to have be part of this query.
 const _getByEntityId = (fields, options, entityId) => sql`
```

Comment:
Objective of the change in this file is to get entity.delete and entity.restore logs as well

Reply:
This query change looks good to me. I remember working on this query in https://github.com/getodk/central-backend/pull/1126 because it was previously very slow. I like your solution better (matches the other audit queries better) and it doesn't feel slow (using my dev database that has 4M+ entities in it.